app/views/posts/index.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="topnav">
+  <%= link_to image_tag("hacker-logo.png", action: 'index', size: "22x22", alt: "Logo")%>
```

Comment:
probably best to use the name of the project instead of `Logo` here for the `alt` tag. for a screenreader, hearing "Logo" doesn't really give you much information. it would be like searching for restaurants in google maps and all of the results just say "restaurant" instead of the name of the restaurant.